db/seeds.rb

```diff
@@ -7,14 +7,15 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 ads_data = []
+number_of_ads = 500
 
 100.times do
```

Comment:
You forgot to update this one
```suggestion
number_of_ads.times do
```

Reply:
Thanks!